rc/tools/clipboard.kak

```diff
@@ -0,0 +1,94 @@
+declare-option -docstring %{
+    The shell command to which data will be piped to copy to the system clipboard
+} str clipboardcmd_copy %sh{
+    for cmd in "xsel -i" "pbcopy" "wl-copy -p"; do
+        readonly program="${cmd%% *}"
```

Comment:
I don't think `readonly` is POSIX (at least, it's not in `dash`).

Reply:
Hmm I can see it in the [OpenGroup publication](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_23), but I suppose if it doesn't work on Dash it will have to go.

I'll have to remove it where there too, since I thought this keyword was portable and started using it everywhere:

https://github.com/mawww/kakoune/blob/df7b33bc7b9da897d1d0127b54ae1e629d2334ff/rc/detection/modeline.kak#L63-L64

Reply:
I'm sorry, I've made a mistake - I'm sure `readonly` wasn't in the dash manpage when I checked yesterday, but it's definitely there today, and testing it does seem to work as intended:

    /bin/dash: 3: foo: is read only